clippercard/parser.py

```diff
@@ -129,6 +129,11 @@ def parse_card_products(card_soup):
         for value_node in blings:
             name = list(value_node.previous_siblings)[1].get_text().strip(':')
             products.append(CardProduct(name=name, value=value_node.get_text()))
+        caltrain = card_section.find_all('div', text=re.compile('^Valid till'))
+        for value_train in caltrain:
+            name = list(value_train.previous_siblings)[1].get_text().strip(':')
```

Comment:
for my own learning, could you paste an example the Caltrain-relevant html snippet that you're parsing?

Reply:
https://gist.github.com/drahamim/210b6aadad668b1a0f6a03c7f21e8dd6